### PR TITLE
clang-tidy fixes

### PR DESCRIPTION
**Pull request by lilleyse**
Follow up to https://github.com/CesiumGS/cesium-omniverse/pull/376 - the clangd extension shows clang-tidy errors so I went ahead and fixed them.

### CODE REVIEW

File: .clang-tidy
Comment (mattelser) [inline .clang-tidy:1]: thank you for removing "easily swappable parameters"!

File: src/core/include/cesium/omniverse/OmniTileset.h [hunk 1]
```diff
@@ -11,14 +11,12 @@
 #include <vector>
 
 namespace Cesium3DTilesSelection {
-struct ImageCesium;
```
Comment (mattelser) [inline src/core/include/cesium/omniverse/OmniTileset.h:14]: Is this removed because `ImageCesium` is available elsewhere?

**Review by mattelser — APPROVED** [review]
assuming that the `ImageCesium` comment above is harmless, lgtm

File: src/core/include/cesium/omniverse/OmniTileset.h (hunk 1 above)
Comment (lilleyse) [reply src/core/include/cesium/omniverse/OmniTileset.h:14]: Yeah, other files are including / forward declaring `ImageCesium`. At one point `OmniTileset.h` must have been using it.

**Pull request merged**